README.md

```diff
@@ -77,6 +77,20 @@ Current ghaf-infra targets:
 ╘═══════════════╧═══════════════╧══════════════╛
 ```
 
+In case `hostname` is not directly accessible for your current `$USER`, use `~/.ssh/config` to specify the ssh connection details such as username, port, or key file used to access the specific host.
+
+As an example, to access host `51.12.56.79` with a specific username and key, you would add the following to `~/.ssh/config`:
+
+```
+$ cat ~/.ssh/config
+Host 51.12.56.79
+    HostName 51.12.56.79
+    User my_other_user_name
```

Comment:
Replace "my_other_user_name" with something that makes clear that it's the username on the other end of the ssh connection. Maybe "my_remote_user_name"

Reply:
Fixed in https://github.com/tiiuae/ghaf-infra/commit/6ce2a72705957278d77857fcd6344821ab1b99be